tests/Unit/ProxyClient/VarnishTest.php

```diff
@@ -109,7 +109,7 @@ public function testTagsHeaders()
             \Mockery::on(
                 function (RequestInterface $request) {
                     $this->assertEquals('BAN', $request->getMethod());
-                    $this->assertEquals('(mytag|othertag)(,.+)?$', $request->getHeaderLine('X-Cache-Tags'));
+                    $this->assertEquals('(mytag|othertag)(,|$))?$', $request->getHeaderLine('X-Cache-Tags'));
```

Comment:
there must be no more question mark. it must either be followed by , or |. the code is right but i think the assumption is wrong.